backend/routes/index.js

```diff
@@ -0,0 +1,174 @@
+var express = require('express');
+var router = express.Router();
+
+//Using request for super simple api requests
+var request = require('request');
+
+//Read our keys.json file
+var fs = require("fs");
+
+//Parse our keys.json
+keys = JSON.parse(fs.readFileSync('keys.json', 'utf8'));
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  res.render('index', { title: 'I am a Hamburger' });
```

Comment:
OHHHHHHH I forgot I put that in there. Yeah julian and I have a joke, similar to googl'es teapot:

http://www.error418.net/

And we can definitely add a return of 418 haha